<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1271</width>
      <height>691</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_view">
     <attribute name="title">
      <string>查看</string>
     </attribute>
     <widget class="QWidget" name="widget_show" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1271</width>
        <height>641</height>
       </rect>
      </property>
      <widget class="QListWidget" name="listWidget_studentNumber">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>190</y>
         <width>211</width>
         <height>381</height>
        </rect>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_chooseItem">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>190</y>
         <width>601</width>
         <height>381</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_show">
       <property name="geometry">
        <rect>
         <x>970</x>
         <y>80</y>
         <width>61</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>查看</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>80</y>
         <width>63</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>选择班级</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>80</y>
         <width>63</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>选择年级</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_grade">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>70</y>
         <width>42</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_class">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>70</y>
         <width>42</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_edit">
     <attribute name="title">
      <string>编辑</string>
     </attribute>
     <widget class="QLabel" name="label_number">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>100</y>
        <width>63</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>选择数量</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_number">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>100</y>
        <width>42</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_number">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>131</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>340</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
